decidim-bulletin-board-app/app/graphql/types/query_type.rb

```diff
@@ -29,6 +29,17 @@ def elections
       Election.all
     end
 
+    field :election,
+          Types::ElectionType, 
+          null: true,
+          description: "Returns an election given its unique_id" do
+      argument :unique_id, String, required: true
```

Comment:
We are using the `unique_id` as the `id` for the whole graphql interface. So, I would prefer to keep calling it `id` in this endpoint. But I understand that this could be confusing (as the database `id` is not used and the `unique_id` is exposed as `id`). Do you think we should change this line to match the rest of the app or change it everywhere to make it more simpler (use `unique_id` instead of `id` everywhere)?

Reply:
I'm in favor to use `unique_id` everywhere, as @leio10 commented just to don't generate confusion, as it doesn't refer to the db `id` but to a custom identifier.

Reply:
Well, I have doubts about that. It makes complete sense inside the app, as we don't use the same name for different things. But from the API consumer point of view it's a bit strange, all ids are supposed to be unique, why don't we just call it `id`?
ElectionGuard uses something called `object_id` to store these slug based identifiers. It would be nice to use that, but all Ruby objects already have an `object_id`, so we can't use that name.
Also, something similar could happen with message identifiers. I didn't use `unique_id` because I'm not sure if they are going to be unique, but my idea was to use them as the `id` field. The problem could arise if we have two messages with the same id, so maybe we should revert this to use the autonumeric field.

Reply:
Honestly... I agree with you both 😂 . I understand that maybe this is giving to much details to the API consumer so calling it just `id` is ok for me even if internally we are using the `unique_id`. TBH I used `id` first and then I changed it but now I am having second thoughts. Maybe using another identifier different from `id` and `unique_id` can help?